### DISCUSSION TITLE
Welcome to ctkchart Discussions!

### DISCUSSION THREAD

**Discussion by thisal-d** [thisal-d/ctkchart#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Akascape** [comment]
@Thisal-D Hey bro, the video files you uploaded in the readme are not playing, showing 404 missing.

**Reply by thisal-d** [reply]
Hey thanks for inform bro,🙂. i fixed that ,
